packages/vue-app/template/client.js

```diff
@@ -157,7 +157,7 @@ async function loadAsyncComponents (to, from, next) {
         (Component, instance) => ({ Component, instance })
       )
       // Add a marker on each component that it needs to refresh or not
-      const startLoader = Components.some(({ Component, instance }) => {
+      const startLoader = Components.some(async ({ Component, instance }) => {
```

Comment:
[Array.prototype.sum](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/some#:~:text=Description,some()%20immediately%20returns%20true%20.) expects callback returning boolean not promise. We may use `map` + `Promise.all` + `some` if want to support this :)

Reply:
i'll do